### ISSUE TITLE
some code question

### ISSUE THREAD

**Issue by acefunbil** [adlsn/LoGBNet#1]
Hello, thank you very much for your contribution and for open-sourcing this paper code!

      I encountered an issue when trying to run the code on my own dataset: when I run train.py, the model returns None.
      I checked the corresponding part of the model code as follows:
      def forward(self, x):
          ...
          d1 = self.decoder1(d2 + e1)
          log_features = self.log_module(x)
          gated_output = self.balance_gate(log_features)
          if gated_output is None:
              return None
          combined_features = torch.cat([d1, gated_output], dim=1)
          output = self.aspp(combined_features)
          return output
         It seems that gated_output comes from queue storage, its corresponding original data does not match the current batch data in the regular flow. Was this design intentional?

         I would really appreciate your help and hope I’m not bothering you!
